zap/src/main/java/org/zaproxy/zap/extension/pscan/ExtensionPassiveScan.java

```diff
@@ -262,11 +262,12 @@ private boolean addPluginPassiveScannerImpl(PluginPassiveScanner scanner) {
 
             added = addPassiveScannerImpl(scanner);
 
-            if (hasView()) {
-                getPolicyPanel().getPassiveScanTableModel().addScanner(scanner);
+            if (added) {
+                if (hasView()) {
+                    getPolicyPanel().getPassiveScanTableModel().addScanner(scanner);
+                }
+                LOGGER.info("loaded passive scan rule: {}", scanner.getName());
```

Comment:
```suggestion
                LOGGER.info("Loaded passive scan rule: {}", scanner.getName());
```

Reply:
Copied this from https://github.com/zaproxy/zaproxy/blob/0d3731df514b242ff9c19c5963af2ae341096308/zap/src/main/java/org/zaproxy/zap/extension/pscan/ExtensionPassiveScan.java#L269 But maybe that one can be updated too.